recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+if "%ARCH%"=="32" (
+    set PLATFORM=Win32
+) else (
+    set PLATFORM=x64
+)
+
+set mpfr_root=%cd%
+cd ..
+mkdir mpir\dll\%PLATFORM%\Release
+mkdir mpir\lib\%PLATFORM%\Release
+
+REM copy mpir headers
+xcopy %LIBRARY_INC%\*.h mpir\lib\%PLATFORM%\Release\ /E
+xcopy %LIBRARY_INC%\*.h mpir\dll\%PLATFORM%\Release\ /E
+
+REM copy mpir libraries
+copy %LIBRARY_LIB%\mpir.dll mpir\dll\%PLATFORM%\Release\mpir.dll
+copy %LIBRARY_LIB%\mpir.lib mpir\dll\%PLATFORM%\Release\mpir.lib
+copy %LIBRARY_LIB%\mpir_static.lib mpir\lib\%PLATFORM%\Release\mpir.lib
```

Comment:
These files are in the build directory and not the `%PREFIX%` directory where conda looks for new files and therefore I don't think there'll be any issues. What do you think?